source/helpers/CaseDataConverter.js

```diff
@@ -97,8 +97,13 @@ export const convertAnswersToArray = (data, formQuestions) => {
               const dynamicTagRegex = new RegExp('.+:x');
               newTags = tags.map((tag) => {
                 if (dynamicTagRegex.test(tag)) {
-                  const strArray = tag.split(':');
-                  return `${strArray?.[0] || tag}:${childFieldId}`;
+                  const tagStringList = tag.split(':');
```

Comment:
Is it important to know that the tag list is of strings? Maybe the variable name could be simplyfied to 'tagList'?

Reply:
or just tags?

Reply:
it's not tags, but all strings that a tag contains ie if the tag is **tag:other:x** then the value would be **[tag, other, x]**. So it's a list of strings that the tag contains